scripts/pshmem_symbol_check.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Simple script to check for missing profiling symbols in 
+# SHMEM library. The readelf tool requires the library to 
+# be a shared object.
+#
+# Example usage: pshmem_symbol_check.sh <path to SHMEM library>
+#
+# Exit if any command fails
+set -e
```

Comment:
Is `set -e` needed?  If I run this tool against a library that did not include `--enable-profiling` (accidentally), then I get a silent error after the first `grep`, and it's not clear what went wrong.

Reply:
Thanks for catching that. Will add something for that. 

I added `-e` thinking this is helpful, but definitely not a requirement.

Reply:
@davidozog Let me know if the recent changes look ok to you. Also, do you prefer `set -e` to be removed? I have not yet, but am ok to remove if there is any preference.

Reply:
@wrrobin - Yes, that change is good - thanks!

I'd say it's ok to can keep `set -e` (at least for now).